src/raft.c

```diff
@@ -122,12 +122,14 @@ void entryAttachRaftReq(RedisRaftCtx *rr, raft_entry_t *entry, RaftReq *req)
  */
 void RaftExecuteCommandArray(RedisModuleCtx *ctx,
                              RedisModuleCtx *reply_ctx,
-                             RaftRedisCommandArray *array)
+                             RaftRedisCommandArray *cmds)
 {
     int i;
 
-    for (i = 0; i < array->len; i++) {
-        RaftRedisCommand *c = array->commands[i];
+    HandleAsking(cmds);
```

Comment:
If there are no other usages, I wouldn't create a function for this. Advantages: No need to declare it, no need to add a field to RaftRedisCommandArray (you can have a local bool variable here).

Reply:
as of now, we don't do anything with the asking flag, in the future we will.  we will also have the slot variable that the commands are associated with.

i.e. like https://github.com/RedisLabs/redisraft/pull/293/files#diff-10528055f382929eb163463ff1402e7d59eb4fbe95ec1d005b52aaf2b148d2b9R479-R480

Reply:
okay